libr/core/cmd_write.c

```diff
@@ -629,6 +630,59 @@ static bool cmd_wfs(RCore *core, const char *input) {
 	return true;
 }
 
+static bool cmd_wfs(RCore *core, const char *input) {
+	char *str = strdup (input);
+	if (str[1] == ' ') {
```

Comment:
too many nested `if`s :/ they make the code unreadable.

Use:
```C
if (str[1] != ' ') {
   eprintf ("Usage wfs host:port [sz]\n");
   goto err;
}
....
if (!port) {
   eprintf ("Usage wts host:port [sz]\n");
   goto err;
}
...
if (!r_socket_listen (s, port, NULL)) {
   eprintf ("Cannot connect\n");
   goto err_socket;
}
...
```

Reply:
done